pkgs/data/icons/rose-pine/default.nix

```
@@ -0,0 +1,38 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation rec {
```

```suggestion
{ stdenvNoCC
, lib
, fetchFromGitHub
, gtk
}:

stdenvNoCC.mkDerivation rec {
```

Usually this is the formatting I would use here, suggested by `nixpkgs-fmt`.

Also use `stdenvNoCC`, because we do not need a complete compiler toolchain for this package.